src/theme/header.php

```diff
@@ -32,41 +36,39 @@
+  <link rel="icon" type="image/jpeg" sizes="512x512" href="<?php echo $template_url; ?>/images/icon.png">
+  <link rel="apple-touch-icon" type="image/jpeg" sizes="512x512" href="<?php echo $template_url; ?>/images/icon.png">
 
-  <title>Aimless Stack Pointers</title>
+  <title><?php echo bloginfo( 'name' ); ?></title>
   <style>
   <?php
-    include(dirname(__FILE__).'/critical.css');
-    include(dirname(__FILE__).'/components/header_critical.css');
-    include(dirname(__FILE__).'/components/footer_critical.css');
-    include(dirname(__FILE__).'/components/article_critical.css');
-    include(dirname(__FILE__).'/components/ribbon_critical.css');
-    include(dirname(__FILE__).'/components/comments_critical.css');
+    include_once dirname( __FILE__ ) . '/critical.css';
+    include_once dirname( __FILE__ ) . '/components/header_critical.css';
+    include_once dirname( __FILE__ ) . '/components/footer_critical.css';
+    include_once dirname( __FILE__ ) . '/components/article_critical.css';
+    include_once dirname( __FILE__ ) . '/components/ribbon_critical.css';
+    include_once dirname( __FILE__ ) . '/components/comments_critical.css';
   ?>
   </style>
   <noscript class="lazyload">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/lazy.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/fonts.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/footer_lazy.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/header_lazy.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/article_lazy.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/ribbon_lazy.css">
-    <link rel="stylesheet" href="<?php echo get_bloginfo('template_url'); ?>/components/comments_lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/fonts.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/footer_lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/header_lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/article_lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/ribbon_lazy.css">
+    <link rel="stylesheet" href="<?php echo $template_url; ?>/components/comments_lazy.css">
   </noscript>
 </head>
 <body>
-  <header class="hero <?php echo is_single()?'single':''; ?>">
+  <header class="hero<?php if ( is_single() ) : ?> single<?php endif; ?>">
     <div class="wrapper">
       <a href="<?php echo home_url(); ?>" class="ribbon ribbon--btt">
-        Aimless<br>
-        Stack<br>
-        Pointer
+        <?php echo bloginfo( 'name' ); ?>
```

Comment:
Yes, but you can use filter to do it.

```PHP
add_filter( 'option_blogname', function() {
  return 'Aimless<br>Stack<br>Pointer';
} );
```

https://developer.wordpress.org/reference/functions/get_option/#source
Line: 141

Reply:
Where would I add that call? In `functions.php`? Because what’s the point of making the title dynamic if I am just going to hard-override it? (Also, the `<br>`s get escaped)

Reply:
Yes, in the `functions.php`, but if I want to edit via admin?